hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/TestDatanodeConfiguration.java

```diff
@@ -274,4 +274,12 @@ static void assertWaitTimeMin(TimeDuration expected,
     assertEquals(expected, t,
         RaftServerConfigKeys.Log.Appender.WAIT_TIME_MIN_KEY);
   }
+
+  @Test
+  public void defaultMinFreeSpacePercentIsValid() {
+    OzoneConfiguration conf = new OzoneConfiguration();
+    conf.unset(DatanodeConfiguration.HDDS_DATANODE_VOLUME_MIN_FREE_SPACE_PERCENT); // ensure default
+    DatanodeConfiguration subject = conf.getObject(DatanodeConfiguration.class);
+    assertEquals(0.001f, subject.getMinFreeSpaceRatio(), 1e-6);
+  }
```

Comment:
This is already covered by:

https://github.com/apache/ozone/blob/4de2f10781377d506daf869aa659c7a583e30f10/hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/TestDatanodeConfiguration.java#L181

so we don't need to add a new test method.  Both old and new assertions pass without the change of `defaultValue` in `DatanodeConfiguration`.  That's because before this change the value is set by `validateMinFreeSpace()`, which emits the warning.  To test for the lack of warning, please capture log output and use its content for the assertion in the existing test case `isCreatedWitDefaultValues`.